rust-toolchain.toml

```diff
@@ -0,0 +1,6 @@
+[toolchain]
+channel = "nightly"
+components = ["clippy", "rustfmt"]
+targets = [
+    "wasm32-unknown-unknown"
```

Comment:
wasm32-wasi?

Reply:
There's nothing we currently build for WASI here